dream_layer_backend/dream_layer.py

```
@@ -1,492 +1,85 @@
+
+        # Send to ComfyUI
+        response = requests.post(COMFY_API_URL, json=graph)
+        response.raise_for_status()
+
+        return jsonify({"status": "success", "message": "Prompt submitted to ComfyUI"})
     except Exception as e:
-        return jsonify({
-            "status": "error",
-            "message": f"Failed to fetch ControlNet models: {str(e)}"
-        }), 500
+        print(f"[ERROR] Txt2Img: Failed to submit prompt to ComfyUI", str(e))
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)